web/src/features/fireBehaviourCalculator/validation.ts

```diff
@@ -7,7 +7,10 @@ import { FBCInputRow } from './components/FBCInputGrid'
  * @param row the row to validate
  * @returns false if valid, true otherwise
  */
-export const grassCureNotSetForGrassType = (row: FBCInputRow) => {
+export const grassCureNotSetForGrassType = (row: FBCInputRow): boolean => {
+  if (isUndefined(row)) {
+    return false
+  }
   if (row.fuelType === 'o1a' || row.fuelType === 'o1b') {
```

Comment:
at some point (I'm not sure how high a priority it is in relation to the other things we're working on right now), but we should add more validation about grass curing values. There's a hover-over text message on the grass cure field saying that values must be between 0-100, but this isn't actually enforced. 

By setting grass curing to be a negative value (with a grass fuel type), the resulting HFI is 0.0
Similarly, I tested out what the HFI result is for a grass cure of 100%, and then calculated again with grass cure of 110%, and the HFI was higher.

Reply:
on further thought, I think it makes more sense to just create a bug ticket for this. We should be adding the validation on the API, too

Reply:
I'll limit it to max = 100